### PR TITLE
Xcode7b2 migration

### LINKED ISSUE #325: Can't install Quick via Carthage under Xcode 7?
I created a brand new Swift project in Xcode 7 and followed the [installation instructions](https://github.com/Quick/Quick/blob/swift-2.0/Documentation/InstallingQuick.md). But when I try and `import Quick` at the top of the default FooTests.swift file, Xcode complains `No such module 'Quick'`.

Has anyone personally had success importing Quick as a framework in Xcode 7 via Carthage?

### PR DESCRIPTION
**Pull request by jeffh**
Ran through Xcode 7 beta 2's Swift Syntax Migrator.

Fixes support for carthage too. Fixes #325.

### CODE REVIEW

File: Quick/DSL/World+DSL.swift [hunk 1]
```diff
@@ -97,6 +97,6 @@ extension World {
     }
 
     internal func pending(description: String, closure: () -> ()) {
-        print("Pending: \(description)")
+        print("Pending: \(description)", appendNewline: false)
```
Comment (modocache) [inline Quick/DSL/World+DSL.swift]: We need to handle pending examples in a nicer way, but for the time being, shouldn't this include a new line? Each pending example should be on its own line in the test output.

**Comment by modocache** [comment]
Looks good to me, just had the one comment around printing pending examples. Feel free to merge!

File: Quick/DSL/World+DSL.swift (hunk 1 above)
Comment (jeffh) [inline Quick/DSL/World+DSL.swift]: Swift 2's print now does newline (true) by default.

**Comment by jeffh** [comment]
Thanks for reviewing, @modocache. :+1:

**Pull request merged**